src/main/scala/com/fulcrumgenomics/util/GeneAnnotations.scala

```diff
@@ -114,4 +115,84 @@ object GeneAnnotations {
     require(start <= end, "start is greater than end when creating an exon")
     def length: Int = end - start + 1
   }
+
+  /** Trait that gene biotypes will extends */
+  sealed trait GeneBiotype extends EnumEntry {
+    /** The biotype name in the GFF */
+    def key: String
+    /** True if this biotype represents a protein coding feature */
+    def isCoding: Boolean
+  }
+
+  /** Enum to represent gene biotypes in a GFF */
+  object GeneBiotype extends FgBioEnum[GeneBiotype] {
+    /** All values of the GeneBiotype Enum */
+    def values: IndexedSeq[GeneBiotype] = findValues
+
+    /** Allows GeneBiotypes to be built from the Enum name of the biotype name from the GFF */
+    override def apply(str: String): GeneBiotype = values.find(_.key == str).getOrElse(super.apply(str))
```

Comment:
The key is so close to being able to be inferred from the class name!

Reply:
I know.  I'm almost tempted to rename the members to match the values in the GFF.  Any thought on whether it is better to have scala-style names or mirror the values in the GFF?